tests/test_base_templates.py

```diff
@@ -245,6 +247,81 @@ def test_get_column_datatype_query():
     assert query == expected
 
 
+def test_get_create_table_from_union():
+    expected = """CREATE TABLE union_table AS
+SELECT
+foo,
+bar,
+baz
```

Comment:
these :should: be indented but SQLFluff was insistent; i decided not to fight it.